webui/react/src/components/TaskFilter.tsx

```diff
@@ -2,146 +2,113 @@ import { Select } from 'antd';
 import { SelectValue } from 'antd/es/select';
 import React, { useCallback, useMemo } from 'react';
 
-import Icon from 'components/Icon';
 import { CommandState, CommandType, RecentTask, RunState, TaskType, User } from 'types';
 import { capitalize } from 'utils/string';
 import { isExperimentTask } from 'utils/task';
-import { commandStateToLabel, runStateToLabel } from 'utils/types';
 
 import IconFilterButtons from './IconFilterButtons';
+import SelectFilter, { ALL_VALUE } from './SelectFilter';
+import StateSelectFilter from './StateSelectFilter';
 import css from './TaskFilter.module.scss';
+import UserSelectFilter from './UserSelectFilter';
 
-const { Option, OptGroup } = Select;
+const { Option } = Select;
 
-export const ALL_VALUE = 'all';
+export { ALL_VALUE };
```

Comment:
blocking: should we export in only in one place either here or select filter?

Reply:
Great idea. I have another PR that's coming up that will do the job of moving `ALL_VALUE` and task filter functions.